case_studies/dc2_cataloging/train_config.yaml

```diff
@@ -34,10 +34,87 @@ variational_factors:
     sample_rearrange: "b ht wt d -> b ht wt 1 d"
     nll_rearrange: "b ht wt 1 d -> b ht wt d"
     nll_gating: is_galaxy
+  # - _target_: bliss.encoder.variational_dist.BivariateNormalFactor
+  #   name: shear
+  #   sample_rearrange: "b ht wt d -> b ht wt 1 d"
+  #   nll_rearrange: "b ht wt 1 d -> b ht wt d"
+  #   nll_gating: is_in_cosmodc2
+  - _target_: bliss.encoder.variational_dist.BivariateNormalFactor
+    name: ellipticity
+    sample_rearrange: "b ht wt d -> b ht wt 1 d"
+    nll_rearrange: "b ht wt 1 d -> b ht wt d"
+    nll_gating: is_in_cosmodc2
+
+my_metrics:
+  detection_performance:
+    _target_: bliss.encoder.metrics.DetectionPerformance
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+  detection_performance_star:
+    _target_: bliss.encoder.metrics.DetectionPerformance
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+    filter_list:
+      - _target_: bliss.encoder.metrics.SourceTypeFilter
+        filter_type: "star"
+  detection_performance_galaxy:
+    _target_: bliss.encoder.metrics.DetectionPerformance
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+    filter_list:
+      - _target_: bliss.encoder.metrics.SourceTypeFilter
+        filter_type: "galaxy"
+  detection_performance_one_source_mask:
+    _target_: bliss.encoder.metrics.DetectionPerformance
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+    filter_list:
+      - _target_: case_studies.dc2_cataloging.utils.metrics.SourceCountFilter
+        filter_source_count: "1m"
+  detection_performance_two_sources_mask:
+    _target_: bliss.encoder.metrics.DetectionPerformance
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+    filter_list:
+      - _target_: case_studies.dc2_cataloging.utils.metrics.SourceCountFilter
+        filter_source_count: "2m"
+  detection_performance_more_than_two_sources_mask:
+    _target_: bliss.encoder.metrics.DetectionPerformance
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+    filter_list:
+      - _target_: case_studies.dc2_cataloging.utils.metrics.SourceCountFilter
+        filter_source_count: "2plus"
+  source_type_accuracy:
+    _target_: bliss.encoder.metrics.SourceTypeAccuracy
+    bin_cutoffs: [200, 400, 600, 800, 1000]
+    bin_type: "Flux"
+  flux_error:
+    _target_: bliss.encoder.metrics.FluxError
+    survey_bands: ${encoder.survey_bands}
+  # shear_error:
+  #   _target_: case_studies.dc2_cataloging.utils.metrics.ShearMSE
+  #   bin_cutoffs: [200, 400, 600, 800, 1000]
+  #   bin_type: "Flux"
+  ellipticity_error:
+    _target_: case_studies.dc2_cataloging.utils.metrics.EllipticityMSE
+    bin_cutoffs: [200, 400, 600, 800, 1000]
```

Comment:
Perhaps we should be specifying everything in terms of magnitude, if those are the units that we'll use for publishing the results

Reply:
change to magnitudes